### PR TITLE
Copy profiles in integration command

### PR DESCRIPTION
**Pull request by therve**

### CODE REVIEW

**Review by olivielpeau — DISMISSED** [review]
LGTM if you've checked it actually works

File: cmd/agent/app/integrations.go [hunk 1]
```diff
@@ -749,6 +749,19 @@ func moveConfigurationFiles(srcFolder string, dstFolder string) error {
 	errorMsg := ""
 	for _, file := range files {
 		filename := file.Name()
+		// Copy SNMP profiles
+		if filename == "profiles" {
+			profileDest := filepath.Join(dstFolder, "profiles")
+			if err = os.MkdirAll(profileDest, os.ModeDir|0755); err != nil {
+				return err
```
Comment (AlexandreYang) [inline cmd/agent/app/integrations.go]: Should we use the same error handling strategy than here? Saving error string to `errorMsg` and `continue`:
https://github.com/DataDog/datadog-agent/blob/master/cmd/agent/app/integrations.go#L758-L767

**Review by AlexandreYang — COMMENTED** [review]
Just one comment, otherwise looks good.

**Comment by ofek** [comment]
Capture

**Review by AlexandreYang — APPROVED** [review]

**Pull request merged**